packages/core/src/node/build.ts

```diff
@@ -80,18 +82,25 @@ export async function renderPages(
     // 2. ssr bundle render failed
     // 3. ssg is disabled
     let render = null;
-    if (enableSSG) {
+    if (ssgConfig) {
       try {
         const { default: ssrExports } = await import(
           pathToFileURL(ssrBundlePath).toString()
         );
         ({ render } = ssrExports as SSRBundleExports);
```

Comment:
This render function gets invoked later to render the page dom. There is also a try catch. Should we also add a throw error there?

Reply:
Nice catch, I missed that.

Updated 😄